nucleus/admin/template/pom.xml

```
@@ -154,4 +154,39 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>UpdateTruststore</id>
```

It's 'trust store' rather than 'truststore' (that's just programmers being lazy), so an extra capital letter
```suggestion
            <id>UpdateTrustStore</id>
```